packages/scm/src/browser/scm-widget.tsx

```diff
@@ -115,7 +118,7 @@ export class ScmWidget extends ScmNavigableListWidget<ScmResource> implements St
         this.scmService.onDidChangeSelectedRepositories(repository => {
             if (repository) {
```

Comment:
when we toggle the widget, old one gets destroyed and new one is created, it looks like for second time this code is never executed

Reply:
I've tested and it works fine